frontend/src/libs/components/profile-settings-sidebar/profile-settings-button/profile-settings-button.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from './styles.module.scss';
+
+type ProfileSettingsButtonProperties = {
+  children: React.ReactNode;
+  icon: string;
+};
+
+const ProfileSettingsButton: React.FC<ProfileSettingsButtonProperties> = (
+  properties,
+) => {
+  const { icon, children } = properties;
+  return (
+    <button className={styles['profile-button']}>
+      <img src={icon} alt="button icon" />
```

Comment:
Let's create an Icon component(check miniproject for some details) but not use FontAwesome, just render .svg